<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appClass</class>
 <widget class="QMainWindow" name="appClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>app</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="title_market">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>title_market</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>101</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="grp_iteminfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Item Information</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="iteminfo_price">
        <property name="text">
         <string>Price:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="iteminfo_uid">
        <property name="text">
         <string>uid: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="iteminfo_bsgid">
        <property name="text">
         <string>bsgid: </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="img_selecteditem">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>75</width>
      <height>75</height>
     </rect>
    </property>
    <property name="text">
     <string>img_selecteditem</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>search items..</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
